napari/layers/points/_points_mouse_bindings.py

```diff
@@ -158,18 +158,18 @@ def _update_drag_vectors_from_event(layer, event):
         # if in 3D, set the drag normal and up directions
         # get the indices of the displayed dimensions
         ndim_world = len(event.position)
-        dims_displayed_data = layer._world_to_data_dims_displayed(
-            dims_displayed=event.dims_displayed, ndim_world=ndim_world
+        layer_dims_displayed = layer._world_to_layer_dims(
+            world_dims=event.dims_displayed, ndim_world=ndim_world
```

Comment:
c.f. discussion above, I'm not 100% that the method which handles slicing down of world dims handles the displayed dims properly, would you mind checking that this change is covered by tests before merge?

Reply:
There's no functional change here. This is just a name change. Agreed there may be dimensional correspondence bugs out there, but I don't think this hefty PR is the place to fix those. I did add some tests to cover `_world_to_layer_dims` in `test_viewer_model.py` to express some of our current logic.